android/java/org/uproxy/tun2socks/Tunnel.java

```diff
@@ -126,6 +126,8 @@ public synchronized void stop() {
 
   private static final String VPN_INTERFACE_NETMASK = "255.255.255.0";
   private static final int VPN_INTERFACE_MTU = 1500;
+  private static final String DNS_RESOLVER_IP = "8.8.8.8";
```

Comment:
My thinking was that allowing the client to intelligently select among several options would allow more resilience to degradation in service quality (or outright blocking) of any one of them (more baskets for our eggs, so to speak). Since it sounds like it might not be YAGNI, I opened #16 to track this.